src/tree.ts

```diff
@@ -460,14 +512,18 @@ export class ProjectOutlineProvider implements vscode.TreeDataProvider<BaseNode>
 
   getChildren(node?: BaseNode): BaseNode[] {
     try {
-      if (!node) {
-        // Request for root node
-        return this._children;
-      } else {
+      if (node) {
         return node.getChildren();
       }
+      // Request for root nodes
+      if (this._folders.size <= 1) {
+        for (const folder of this._folders.values()) {
+          return folder.getChildren();
+        }
+      }
+      return [...this._folders.values()];
     } catch (e) {
-      rollbar.error(localize('error.rendering.children.nodes', 'Error while rendering children nodes'));
+      rollbar.error('Error while rendering children nodes');
```

Comment:
Can you restore the localized string?

Reply:
Restore the localized string?

Reply:
Yes, the code used to be: `localize('error.rendering.children.nodes', 'Error while rendering children nodes')`, but now it is just: `'Error while rendering children nodes'`.  The string should be localized.

Reply:
Ah read it wrong. Sure I'll change it back. Not sure why I changed it thought...